test-suite/output/PrintNotation.out

```diff
@@ -0,0 +1,129 @@
+Notation "_ $ _" at level 123 with arguments constr at next level, constr
+at next level, No associativity defined.
```

Comment:
Is "No associativity defined" meant to be final, or is it a placeholder? If the former, I guess it should not be capitalized.

Reply:
So there are three values: left, right, no and these are put in a `option`. This means we have four values. However confusingly `no` isn't treated correctly causing it to be defaulted to `None`. This means that when we print it we only see what we print for `None` which is `No associativity defined`.

Since the morally correct fix for this will require some major clean up in notations as @herbelin indicated, I believe the best thing we can do is replace "No associativity defined" with "no associativity".